utilities_common/common.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python -u
+# -*- coding: utf-8 -*-
+
+__all__ = ['log_info', 'log_error', 'log_warning', 'log_debug',
+        'do_exit', 'get_configdb_data', 
+        'AbbreviationGroup', 'get_hostname']
+
+import sys
+import os
+import syslog
+import click
+import inspect
+from swsssdk import ConfigDBConnector
+
+SYSLOG_IDENTIFIER = "config"
+
+def get_hostname():
+    return os.uname()[1]
```

Comment:
This is not CLI-specific. This is better suited for sonic-py-common.

Reply:
Moved to sonic_py_common/device_info.py